Makefile

```diff
@@ -1,10 +1,10 @@
 JBUILDER ?= jbuilder
 
 all:
-	$(JBUILDER) build --dev
+	$(JBUILDER) build
 
 tests:
-	$(JBUILDER) runtest --dev
+	$(JBUILDER) runtest
```

Comment:
Why did you remove `--dev` here? You don't like some of the warnings?

Reply:
Yes. I remember being annoyed by warning 27. There may be others.
> 27 Innocuous unused variable: unused variable that is not bound with let nor as, and doesn’t start with an underscore (_) character.

I want to write patterns like `Node (loc, x, _)` without having to write `_loc` in case I don't use `loc` later. I don't want to write `Node (_, x, _)` either because the day I need to produce an error that uses `loc`, I don't know where to find it. I also don't want in the same pattern-matching to haves cases like `Node (_loc, x, _)` and other cases like `Other (loc, y)`.